src/commands/android/subcommands/uninstall/system-image.ts

```diff
@@ -0,0 +1,94 @@
+import colors from 'ansi-colors';
+import inquirer from 'inquirer';
+
+import Logger from '../../../../logger';
+import {Platform} from '../../interfaces';
+import {execBinarySync} from '../../utils/sdk';
+import {getBinaryLocation} from '../../utils/common';
+import {getInstalledSystemImages, showMissingBinaryHelp} from '../common';
+
+export async function deleteSystemImage(sdkRoot: string, platform: Platform): Promise<boolean> {
+  try {
+    const sdkmanagerLocation = getBinaryLocation(sdkRoot, platform, 'sdkmanager', true);
+    if (!sdkmanagerLocation) {
+      showMissingBinaryHelp('sdkmanager');
+
+      return false;
+    }
+    const installedImages: string[] = await getInstalledSystemImages(sdkmanagerLocation, platform);
+    if (!installedImages.length) {
+      return false;
+    }
```

Comment:
`getInstalledSystemImages` no longer return just `string[]`.

Also, if we are returning `false` here because no system image is already installed, we should call that out before returning.